src/cdk-experimental/popover-edit/popover-edit-position-strategy-factory.ts

```diff
@@ -24,19 +36,39 @@ export abstract class PopoverEditPositionStrategyFactory {
  */
 @Injectable()
 export class DefaultPopoverEditPositionStrategyFactory extends PopoverEditPositionStrategyFactory {
-  constructor(protected readonly overlay: Overlay) {
+  constructor(protected readonly direction: Directionality, protected readonly overlay: Overlay) {
```

Comment:
The `readonly` part here seems a little redudant. If somebody decides to change these, they either know what they're doing or they're shooting themselves in the foot.